group-coordinator/src/main/java/org/apache/kafka/coordinator/group/classic/ClassicGroup.java

```diff
@@ -1400,7 +1400,7 @@ public static ClassicGroup fromConsumerGroup(
                         Optional.ofNullable(member.instanceId()),
                         member.clientId(),
                         member.clientHost(),
-                        member.rebalanceTimeoutMs(),
+                        member.classicProtocolSessionTimeout().get(),
```

Comment:
This is incorrect, right? It should be the following line. I also wonder if we should use get or default to `consumerGroupSessionTimeoutMs`.

Reply:
Ah yes, thanks for the catch!!